application/src/main/java/org/thingsboard/server/actors/tenant/TenantActor.java

```diff
@@ -249,6 +259,10 @@ private void onComponentLifecycleMsg(ComponentLifecycleMsg msg) {
                 Edge edge = systemContext.getEdgeService().findEdgeById(tenantId, edgeId);
                 edgeRpcService.updateEdge(tenantId, edge);
             }
+        } else if (msg.getEntityId().getEntityType() == EntityType.DEVICE && ComponentLifecycleEvent.DELETED == msg.getEvent()) {
```

Comment:
Please rewrite all comparisons in this method to be consistent. For example:

Method contains both `msg.getEntityId().getEntityType().equals(EntityType.API_USAGE_STATE)` and `msg.getEntityId().getEntityType() == EntityType.EDGE`.
Also, method contains both `msg.getEvent() == ComponentLifecycleEvent.DELETED` and `ComponentLifecycleEvent.DELETED == msg.getEvent()`